api/package.json

```diff
@@ -12,7 +12,8 @@
     "clean": "rm -rf -v dist/*",
     "validate-template": "aws cloudformation validate-template --template-body file://template.yaml",
     "describe-stack": "aws cloudformation describe-stacks --stack-name $npm_package_name-${STAGE:-dev}",
-    "dev-server": "nodemon --exec \"sam local start-api -n env.json\" --watch dist"
+    "dev-server": "nodemon --exec \"sam local start-api -n env.json\" --watch dist",
+    "lint": "./node_modules/tslint/bin/tslint --project tsconfig.json 'src/**/*.ts' 'src/**/*.tsx'"
```

Comment:
If you take a look at `frontend/node_modules/.bin/`, you'll see that it has `tslint` present. This dir is added to the path when you use `npm run`, meaning you can just call the binary by its name.